lib/fluent/plugin/in_monitor_agent.rb

```diff
@@ -364,6 +368,13 @@ def get_monitor_info(pe, opts={})
           }
         end
         obj['instance_variables'] = iv
+      elsif ivars = opts[:ivars]
+        iv = {}
+        ivars.each {|name|
+          iname = :"@#{name}"
```

Comment:
What does 'garbage' mean?
`:"@#{name}"` is dynamic symbol genaration via string, not static literal.